fault_scarp/landlab-fault-scarp.ipynb

```diff
@@ -103,15 +97,15 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Let's take a look at the grid we've created. To do so, we'll use the Matplotlib graphics library (imported under the name `plt`). We also have to tell the iPython Notebook to display plots right here on the page."
+    "The above line of code creates space in memory to store 1,000 floating-point values, which will represent the elevation of the land surface at each of our 1,000 grid nodes.\n",
+    "\n",
+    "Let's take a look at the grid we've created. To do so, we'll use the Matplotlib graphics library (imported under the name `plt`). We also have to tell the Notebook to display plots right here on the page."
```

Comment:
The order of explanation of matplotlib import and the magic plot display is reverse of the code block. I suggest switching the code commands or explanation.

Reply:
Done, thanks for catching that